gammapy/irf/energy_dispersion.py

```diff
@@ -116,47 +116,37 @@ def pdf_in_safe_range(self, lo_threshold, hi_threshold):
         return data
 
     @classmethod
-    def from_gauss(cls, e_true, e_reco, sigma=0.2, pdf_threshold=1e-6):
+    def from_gauss(cls, e_true, e_reco, sigma, bias, pdf_threshold=1e-6):
         """Create Gaussian `EnergyDispersion` matrix.
 
-        The output matrix will be Gaussian in log(e_true / e_reco)
-
-        TODO: extend to have a vector of bias various true energies.
-        TODO: extend to have vector of  resolution for various true energies.
-        TODO: give formula: Gaussian in log(e_reco)
-        TODO: add option to add poisson noise
+        Calls :func:`~gammapy.irf.EnergyDispersion2D.from_gauss`
 
         Parameters
         ----------
         e_true : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
             Bin edges of true energy axis
         e_reco : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
             Bin edges of reconstructed energy axis
-        sigma : float, optional
+        bias : float or `~numpy.ndarray`
+            Center of Gaussian energy dispersion, bias
```

Comment:
This description of the `bias` argument is a bit vague, I think people will struggle to understand it.

Discussing the definition / sign might take a while:
https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/issues/93

For now, my suggestion would be to do something like "see definition in `get_bias`" where the formula is given how the bias is defined from here and the other `from_gauss` function that has a `bias` parameter.